lib/rules/jsx-sort-default-props.js

```diff
@@ -16,6 +16,7 @@ const propWrapperUtil = require('../util/propWrapper');
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
would sorting rules not be "layout"?
```suggestion
    type: 'layout',
```

Reply:
hmm, fair point.